ztools/http/client.go

```diff
@@ -613,6 +613,10 @@ func (c *Client) Do(req *Request) (resp *Response, err error) {
 					timeout: true,
 				}
 			}
+			if err == io.EOF {
```

Comment:
Wait, do we need this change? The HTTP lib should ideally behave the same, we should catch any differences zgrab has between HTTP at the ZGrab logic. Doesn't the change to `grabber.go` fix that?